synapse/cores/common.py

```diff
@@ -141,6 +141,8 @@ def __init__(self, link, **conf):
         self.loadbus.on('core:save:del:rows:by:prop', self._loadDelRowsByProp)
         self.loadbus.on('core:save:set:rows:by:idprop', self._loadSetRowsByIdProp)
         self.loadbus.on('core:save:del:rows:by:idprop', self._loadDelRowsByIdProp)
+        self.loadbus.on('syn:core:blob:set', self._onSetBlobValu)
```

Comment:
For internals like this, i'd rather we just call this function directly before firing the event.  it could be possible to get into weird circumstances ( possibly via the @on() decorator ) where other callbacks which expect the storage layer update to be complete could run before these.  Better to just call explicitly and leave the bus for external consumers